tests/x11/yast2_snapper.pm

```diff
@@ -135,13 +136,25 @@ sub run() {
 
 sub post_fail_hook {
     my ($self) = @_;
+    # snapper actions can put the system under quite some load so we want to
+    # give some more time, e.g. for login in the consoles
+    my $previous_timeout_scale = get_var('TIMEOUT_SCALE', 1) * 10;
```

Comment:
in backend we have function scale_timeout .... not sure , but probably will be more correct to call it here ? Anyway I don't like some magic formula which you need to support in one single place